### PR TITLE
make plugin global to python code

### PR DESCRIPTION
**Pull request by AlanJAS**
Fix typo of previous pull request.
Do plugin names globals in the pyexported python file.

### CODE REVIEW

File: TurtleArt/taexportpython.py [hunk 1]
```diff
@@ -23,6 +23,7 @@
 import ast
 from gettext import gettext as _
 from os import linesep
+from os import path,pardir
```
Comment (walterbender) [inline TurtleArt/taexportpython.py]: Please add space after comma

File: TurtleArt/taexportpython.py [hunk 2]
```diff
@@ -46,18 +47,21 @@
...
 import os, sys, dbus
 paths = []
+
 paths.append('../%s.activity')
 paths.append(os.path.expanduser('~') + '/Activities/%s.activity')
 paths.append('/usr/share/sugar/activities/%s.activity')
 paths.append('/usr/local/share/sugar/activities/%s.activity')
 paths.append(
     '/home/broot/sugar-build/build/install/share/sugar/activities/%s.activity')
+"""+"paths.append('%s')"%path.abspath(path.join(path.dirname(__file__), pardir))+"""
```
Comment (walterbender) [inline TurtleArt/taexportpython.py]: What is this?

File: TurtleArt/taexportpython.py [hunk 3]
```diff
@@ -46,18 +47,21 @@
...
 
 flag = False
 for path in paths:
     for activity in ['TurtleBots', 'TurtleBlocks']:
-        p = path % activity
-        if os.path.exists(p):
+        p = (path % activity) if "%" in path else path
```
Comment (walterbender) [inline TurtleArt/taexportpython.py]: Please use single quotes

File: TurtleArt/taexportpython.py [hunk 4]
```diff
@@ -92,14 +100,14 @@ def %s():
...
+    global global_objects,turtles,canvas,logo
     global_objects = tw.get_global_objects()
-"""
-_ACTION_STACK_PREAMBLE = """\
     turtles = tw.turtles
-    turtle = turtles.get_active_turtle()
     canvas = tw.canvas
     logo = tw.lc
-
+"""
```
Comment (walterbender) [inline TurtleArt/taexportpython.py]: What is happening here? How is this related?

File: TurtleArt/taexportpython.py [hunk 5]
```diff
@@ -128,6 +136,12 @@ def save_python(tw):
             blocks_covered.update(set(block_stack))
 
     snippets = [_SETUP_CODE_START]
+
+    for k in plugins_in_use:
+        snippets.append("%s = None\n" % (k.lower(),))
```
Comment (walterbender) [inline TurtleArt/taexportpython.py]: Please use single quotes whenever possible.

**Comment by walterbender** [comment]
(!0 could you please a separate PR for the typo?
(2) I made a few comments/had a few questions in the plugin patch.

File: TurtleArt/taexportpython.py (hunk 2 above)
Comment (AlanJAS) [inline TurtleArt/taexportpython.py]: This adds the current folder in the path list. Maybe is not neccesary.

File: TurtleArt/taexportpython.py (hunk 4 above)
Comment (AlanJAS) [inline TurtleArt/taexportpython.py]: Only a style change.

**Comment by AlanJAS** [comment]
I dont know how to separate the PR. Maybe temove the plugin conmit of my fork is enough.

**Comment by walterbender** [comment]
It is fine. I'll merge.

On Wed, Jun 8, 2016 at 8:33 AM, Alan Aguiar <notifications@github.com>
wrote:

> I dont know how to separate the PR. Maybe temove the plugin conmit of my
> fork is enough.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/sugarlabs/activity-turtleart-gtk2/pull/26#issuecomment-224575034>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ADz74Tb9GAf1bq-R8FzqeYXejgd-PcO7ks5qJraqgaJpZM4Iwk55>
> .
>



-- 
Walter Bender
Sugar Labs
http://www.sugarlabs.org
<http://www.sugarlabs.org>

**Comment by AlanJAS** [comment]
I can do the corrections of tais in other PR. Or you will fix them?
